modules/vstudio/tests/fs2005/projectsettings.lua

```diff
@@ -153,6 +153,24 @@
 		]]
 	end
 
+	function suite.onVs2019()
+		p.action.set("vs2019")
+		prepare()
+		test.capture [[
+	<PropertyGroup>
+		<Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+		<Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+		<ProjectGuid>{AE61726D-187C-E440-BD07-2556188A6565}</ProjectGuid>
+		<OutputType>Exe</OutputType>
+		<AppDesignerFolder>Properties</AppDesignerFolder>
+		<RootNamespace>MyProject</RootNamespace>
+		<AssemblyName>MyProject</AssemblyName>
+		<TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
Did you confirm this by manually creating the project? It's a bit strange the version didn't change between VS2017 and VS2019.

Reply:
Yea, that's an oversight by me, fixed!